charts/stable/homarr/templates/common.yaml

```diff
@@ -1 +1,11 @@
-{{ include "tc.v1.common.loader.all" . }}
+{{/* Make sure all variables are set properly */}}
+{{- include "tc.v1.common.loader.init" . }}
+
+{{/* Render secrets for homarr */}}
+{{- $secrets := include "homarr.secrets" . | fromYaml -}}
+{{- if $secrets -}}
+  {{- $_ := set .Values.secret "secrets" $secrets -}}
```

Comment:
“secrets” must be homarr-secrets or the other way around

Reply:
Like this? `{{- $_ := set .Values.secret "homarr-secrets" $secrets -}}`

Reply:
It didn't like that for some reason, linting failed. I'd like to note I got the example from fireflyiii where it uses just "secrets" https://github.com/truecharts/charts/blob/ed563c07a19c786bbcbf5ec8917f3403697c08b3/charts/stable/fireflyiii/templates/common.yaml#L7C29-L7C29

Reply:
And should we just set the default port to 7575 for now?

Reply:
> It didn't like that for some reason, linting failed. I'd like to note I got the example from fireflyiii where it uses just "secrets" https://github.com/truecharts/charts/blob/ed563c07a19c786bbcbf5ec8917f3403697c08b3/charts/stable/fireflyiii/templates/common.yaml#L7C29-L7C29

Gotta check ff

Reply:
> And should we just set the default port to 7575 for now?

Yea go ahead. We can look at it again later